spacepy/coordinates.py

```diff
@@ -391,3 +397,81 @@ def append(self, other):
         data.extend(list(otherdata.data))
         newobj = Coords(data, dtype=self.dtype, carsph=self.carsph)
         return newobj
+
+    # -----------------------------------------------
+    def to_skycoord(self):
+        '''
+        Create an Astropy SkyCoord instance based on this instance
+
+        Returns
+        =======
+        out : Astropy SkyCoord instance
+            This coordinate as an Astropy SkyCoord
+
+        Notes
+        =====
+        This method requires Astropy to be installed.
+
+        This method uses the GEO coordinate frame as the common frame between the two libraries.
+        '''
+        try:
+            from astropy.coordinates import SkyCoord
+            from astropy.time import Time
+        except ImportError as e:
+            raise e.__class__("This method requires Astropy to be installed.")
+
+        # Convert to GEO (i.e., Astropy's ITRS)
+        coord = self.convert('GEO', 'car')
+
+        # Convert the data to be in units of m
+        data = coord.data * self.Re
+
+        # Convert the Ticktock ticks to an Astropy Time instance
+        if coord.ticks is not None:
+            obstime = Time(coord.ticks.ISO, format='isot', scale='utc')
+        else:
+            obstime = None
+
+        return SkyCoord(x=data[:, 0], y=data[:, 1], z=data[:, 2],
+                        unit='m', frame='itrs', obstime=obstime)
+
+    # -----------------------------------------------
+    @classmethod
+    def from_skycoord(cls, skycoord):
+        '''
+        Create a Coords instance from an Astropy SkyCoord instance
+
+        Parameters
+        ==========
+        skycoord : Astropy SkyCoord instance
+            The coordinate to be converted
+
+        Returns
+        =======
+        out : Coords instance
+            The converted coordinate
+
+        Notes
+        =====
+        This method requires Astropy to be installed.
+
+        This method uses the GEO coordinate frame as the common frame between the two libraries.
+        '''
+        try:
+            from astropy.coordinates import SkyCoord
+        except ImportError as e:
+            raise e.__class__("This method requires Astropy to be installed.")
```

Comment:
```suggestion
            raise e.__class__("This method requires Astropy to be installed.") from e
```

Reply:
Done

Reply:
Reverted.  SpacePy still tests against Python 2.7, where this is not valid syntax.